src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,27 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends BasePage{
+
+    //private String avatarLocator = "a .avatar";
+    @FindBy(css="a .avatar")
+    private WebElement avatarLocator;
+
+
+
+    public HomePage(WebDriver givenDriver){
+        super(givenDriver);
+    }
+
+    public WebElement GetUserAvatar() {
+       return findElement(avatarLocator);
+    }
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._

Reply:
ok, i've deleted it